dev/TEAL.md

```diff
@@ -102,6 +102,9 @@ For two-argument ops, `A` is the previous element on the stack and `B` is the la
 | `^` | A bitwise-xor B |
 | `~` | bitwise invert value X |
 | `mulw` | A times B out to 128-bit long result as low (top) and high uint64 values on the stack |
+| `cons` | pop two byte strings A and B and join them, push the result |
```

Comment:
Very small nit: Could we rename it to `concat` or `append`? It seems to fit slightly better. (I associate `cons` with something else.)